File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/core/noteslist/NotesListFragment.java
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.core.noteslist;

import com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.EntitiesListWithSearchFragment;
import com.github.valhallalabs.laverna.persistent.entity.Note;

/**
 * @author Andrii Bei <psihey1@gmail.com>
 */

public interface NotesListFragment extends EntitiesListWithSearchFragment {

    void showSelectedNote(Note note);
}

File: app/src/main/java/com/github/android/lvrn/lvrnproject/view/fragment/entitieslist/core/noteslist/NotesListFragment.kt
package com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.core.noteslist

import com.github.android.lvrn.lvrnproject.view.fragment.entitieslist.EntitiesListWithSearchFragment
import com.github.valhallalabs.laverna.persistent.entity.Note

interface NotesListFragment : EntitiesListWithSearchFragment {

    fun showSelectedNote(note: Note)
}